src/common/lib/python/common_extension.c

```diff
@@ -99,7 +99,7 @@ static long PyObjectID_hash(PyObjectID *self) {
 }
 
 static PyObject *PyObjectID_repr(PyObjectID *self) {
-  int hex_length = 2 * UNIQUE_ID_SIZE + 1;
+  enum { hex_length = 2 * UNIQUE_ID_SIZE + 1 };
```

Comment:
Non-constant array sizes aren't valid in pre-C99(?) and gave me errors. And they don't compile to the same code post-C99.